src/createInput.ts

```diff
@@ -32,9 +32,7 @@ const createInput_ = <T>(defaultValue: T = empty) => {
     if (defaultValue !== empty) {
       subject.next((current.latest = defaultValue))
     }
-    const source = shareLatest(true, () => cache.delete(key))(
-      subject,
-    ) as Observable<T>
+    const source = shareLatest(subject, () => cache.delete(key))
```

Comment:
All good 👍  
I was thinking - the original had complete:true so maybe it would need takeUntilComplete? => Nope, the subject by definition of `createInput` never completes.

Reply:
True story: since it was indifferent whether to use `true` or `false`, I decided to go with `true` because it had less characters :grimacing:  (smaller bunlde-size :see_no_evil:)